OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/Appearances.java

```diff
@@ -40,8 +39,19 @@ public static void applyDrawableToImageView(Drawable drawable, ImageView view, i
 
     public static String getSensorDisplayName(SensorAppearance appearance, Context context) {
         String units = appearance.getUnits(context);
+        /*
+        bad fix due to an Android bug: if format is "%1s (%2s)" and %2s value length is 1, it adds
```

Comment:
String.format is behaving appropriately. The problem was that the format string should have used %1$s and %2$s.

Reply:
Ok, I reverted it back, but I need someone else to fix the problems in all strings.xml files. I didn't do it because my sensation is that those files are generated with some tools, thus I avoided changing directly them. Should I do it?